cypress/e2e/visit-base-page.cy.js

```diff
@@ -1,6 +1,46 @@
-describe('Sample describe', () => {
+describe("Authentication", () => {
+  it("Login with standard user", () => {
+    cy.visit("/").url().should("eq", "https://www.saucedemo.com/");
+    cy.get('[data-test="username"]').type("standard_user");
+    cy.get('[data-test="password"]').type("secret_sauce");
+    cy.get('[data-test="login-button"]').click();
```

Comment:
general comment - verify if cookie is properly set up after clicking on login button. use cy.getCookie()